### PR TITLE
Fix issue with detection logic of Centos in install.sh

### PR DESCRIPTION
**Pull request by Evalle**
Signed-off-by: evalle shmarnev@gmail.com

### CODE REVIEW

File: hack/install.sh [hunk 1]
```diff
@@ -109,6 +109,9 @@ do_install() {
 	if [ -z "$lsb_dist" ] && [ -r /etc/fedora-release ]; then
 		lsb_dist='fedora'
 	fi
+    if [ -z "$lsb_dist" ] && [ -r /etc/centos-release ]; then
```
Comment (thaJeztah) [inline hack/install.sh]: the indentation looks off; perhaps you're using spaces instead of tabs?

**Comment by thaJeztah** [comment]
thanks @evalle!

ping @tianon @jfrazelle for hack/install.sh

**Comment by Evalle** [comment]
no problem @thaJeztah 
Issue with space/tab fixed.

File: hack/install.sh (hunk 1 above)
Comment (Evalle) [inline hack/install.sh]: Thank you, I see it now, It is a setting in my vim editor (python-like tabs).

**Comment by icecrime** [comment]
@Evalle You last commit ins't signed, can you please fix it and squash the two? Thanks!

**Comment by Evalle** [comment]
@icecrime done! Thank you for pointing me to this.

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**